src/constants/networkConstants.ts

```diff
@@ -19,22 +19,22 @@ const NetworkConstants = {
     UserSocialLoginCallback: `${BaseCanonizerApiUrl}/api/v3/user/social/callback`,
     CountryCodes: `${BaseCanonizerApiUrl}/api/v3/country/list`,
     UpdateUser: `${BaseCanonizerApiUrl}/api/v3/users/`,
-    ChangePassword: `${BaseCanonizerApiUrl}/api/v3/changepassword`,
+    ChangePassword: `${BaseCanonizerApiUrl}/api/v3/change-password`,
```

Comment:
Haven't we discussed to move the `/api/v3/` into the .env file?

Reply:
Done